host/package.json

```diff
@@ -24,6 +24,7 @@
     "trim": "^1.0.1",
     "trim-newlines": "^4.0.2",
     "typescript": "^4.3.0",
+    "victory": "^36.6.11",
```

Comment:
victory is the D3 based charting library that we've elected to use for the various graphs we'll need in our component. We reviewed Victory, Visx and Recharts and selected Victory after developing a proof of concept. 
See: https://formidable.com/open-source/victory/